python/dgl/heterograph_index.py

```diff
@@ -254,6 +254,21 @@ def copy_to(self, ctx):
         """
         return _CAPI_DGLHeteroCopyTo(self, ctx.device_type, ctx.device_id)
 
+    def pin_memory(self):
```

Comment:
Please update the python doc of pin_memory and pin_memory_, and also the c++ documentation for NDArray.

Reply:
Updated.